ansible/vars/fedora/25.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Could we have `repo_rawhide_enabled: 0` here as well? It would be nice if all `fedora/<version>.yml` files would have all options, but each one set with the right value.

Reply:
What would be the point of adding it to non-rawhide template? It'd just be an unused variable. You could set it to `0`, `1`, or `"This is a value"` and it wouldn't have any effect.

Reply:
Given what I said, I'm removing the `fedora`, `updates` and `updates-testing` from the rawhide variables.. It doesn't make any sense to have them there either.

Reply:
The point would be to explicitly show all options that should be configured in each case. The variable not used is harmless, compared to the "documentation POV".

I'll not NACK the PR because of that, though.

Reply:
My point is, for rawhide, these variables do not need to be and should not be configure. Same thing for branched, except with a different repo. Rawhide is a special case of Fedora, so it's natural to treat it a bit differently.